editor/src/components/inspector/common/layout-property-path-hooks.ts

```diff
@@ -339,16 +339,14 @@ export function usePinToggling(): UsePinTogglingResult {
   const [lastHorizontalProp, setLastHorizontalProp] = React.useState<LayoutPinnedProp | null>(null)
   const [lastVerticalProp, setLastVerticalProp] = React.useState<LayoutPinnedProp | null>(null)
 
-  const pinsInfo: PinsInfo = {
```

Comment:
what was wrong with this guy? is the rules of hooks eslint rule stricter now?

Reply:
pinsInfo changes every render and would break the equality check further on